ArchiSteamFarm/NLog/Logging.cs

```diff
@@ -181,6 +183,10 @@ internal static class Logging {
 				CleanupFileName = false,
 				DeleteOldFileOnStartup = true,
 				FileName = Path.Combine("${currentdir}", SharedInfo.LogFile),
+
+				// For GET /Api/NLog/File ASF API usage on Windows (sigh)
```

Comment:
I think this comment may be improved. Maybe reference the `IOException` when calling `File.ReadAllLines[Async](...)`